<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>640</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>640</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>640</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>800</width>
     <height>594</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>800</width>
     <height>594</height>
    </size>
   </property>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>800</width>
      <height>564</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>800</width>
      <height>564</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>800</width>
      <height>564</height>
     </size>
    </property>
    <property name="tabsClosable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>196</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="layoutZoom">
     <item>
      <widget class="QLabel" name="labelZoom">
       <property name="text">
        <string>Zoom</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="botaoZoomIn">
       <property name="text">
        <string>+</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="botaoZoomOut">
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>0</y>
      <width>272</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="layoutBrilho">
     <item>
      <widget class="QLabel" name="labelBrilho">
       <property name="text">
        <string>Brilho</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="botaoMaisBrilho">
       <property name="text">
        <string>+</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="botaoMenosBrilho">
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="textoNivelDeBrilho">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>0</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="layoutRotacao">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Rotacionar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="rotacionarHorario">
       <property name="text">
        <string>&lt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="rotacionarAntiHorario">
       <property name="text">
        <string>&gt;</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>800</width>
     <height>25</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>800</width>
     <height>25</height>
    </size>
   </property>
   <widget class="QMenu" name="menuAbrir_Imagem">
    <property name="title">
     <string>Arquivo</string>
    </property>
    <addaction name="actionAbrir_Imagem"/>
   </widget>
   <addaction name="menuAbrir_Imagem"/>
  </widget>
  <action name="actionAbrir_Imagem">
   <property name="text">
    <string>Abrir Imagem</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
